content/scout/image-analysis.md

```diff
@@ -202,3 +202,33 @@ reference documentation:
 
 - [`docker scout quickview`](../engine/reference/commandline/scout_quickview.md)
 - [`docker scout cves`](../engine/reference/commandline/scout_cves.md)
+
+## Vulnerability severity assessment
+
+Docker Scout assigns a severity rating to vulnerabilities based on
+vulnerability data from [advisory sources](./advisory-db-sources.md).
+Advisories are ranked and prioritized depending on the type of package that's
+affected by a vulnerability. For example, if a vulnerability affects an OS
+package, the severity level assigned by the distribution maintainer is
+prioritized.
+
+If the preferred advisory source has assigned a severity rating to a CVE, but
+not a CVSS score, Docker Scout falls back to displaying the NIST CVSS score
```

Comment:
It's not just NIST it falls back to, it falls back down the list of priorities until it finds a source that has a CVSS score. Often this is NIST, but not always.